themes/custom/droopler_theme/scss/components/_d_p_side_by_side.scss

```diff
@@ -39,6 +40,7 @@ $d-p-side-by-side-image-background-padding-y: 130px !default;
 
   .text-content {
     padding-bottom: $d-p-side-by-side-padding-y;
+    padding-top: $d-p-side-by-side-padding-x;
```

Comment:
`paddding-x`  w `top` ? to na pewno jest tu potrzebne ? nad tym powinna byc ikonka :thinking:

Reply:
właśnie `x` w sensie poziomy?